tools/libvirt_test.sh

```diff
@@ -82,11 +82,21 @@ BLUEPRINT_FILE=${TEMPDIR}/blueprint.toml
 COMPOSE_START=${TEMPDIR}/compose-start-${IMAGE_KEY}.json
 COMPOSE_INFO=${TEMPDIR}/compose-info-${IMAGE_KEY}.json
 
+WORKSPACE=/tmp/
```

Comment:
Do not override this variable, instead define it only if undefined. Example: `WORKSPACE="${WORKSPACE:-/tmp}"` and put it somewhere at the top where other vars are defined.